a10_octavia/controller/worker/tasks/a10_network_tasks.py

```diff
@@ -757,74 +757,76 @@ def execute(self, vthunder, lb_resource, vrid_list, subnet,
         :return: return the update list of VRID object, If empty the need to remove all VRID
         objects from DB else need update existing ones.
         """
-        vrid_value = CONF.a10_global.vrid
-        prev_vrid_value = vrid_list[0].vrid if vrid_list else None
-        updated_vrid_list = copy.copy(vrid_list)
-        if use_device_flavor:
-            if vthunder_config.vrid_floating_ip:
-                conf_floating_ip = vthunder_config.vrid_floating_ip
+        updated_vrid_list = []
+        if subnet:
```

Comment:
I prefer this way:
```
if not subnet:
    return updated_vrid_list
```

Reply:
I think there is no need to explicitly add this check as we are returning the `updated_vrid_list` at the end of the task in common which will return [] in case of `not subnet`
https://github.com/a10networks/a10-octavia/blob/d1405aefdf00b9db51e8b231d405aadb8c96bbd8/a10_octavia/controller/worker/tasks/a10_network_tasks.py#L831

Reply:
Added this:
```
if not subnet:
    return updated_vrid_list
```